Change route query documentation to be less missleading regarding available function

Mandur opened the pull request:
Current documentation suggest that routes have same functionality as twin and job queries. It is not true as function like IS_ARRAY or IS_OBJECT are not available in the message routes. Hence I changed the sentence to be less misleading. Should We list the gap as well?

azurecla commented:
Hi __@ostrich75__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (larryzh). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ostrich75 commented:
#sign-off

azurecla commented:
Hi __@ostrich75__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (larryzh). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

tysonn commented:
@ostrich75 - See mail about working in the private repo, please

Mandur committed:
Change route query documentation

Current documentation suggest that routes have same functionality as twin and job queries. It is not true as function like IS_ARRAY or IS_OBJECT are not available in the message routes. Hence I changed the sentence to be less misleading. Should We list the gap as well?
